patches/@haqq+provider-ledger-react-native+0.0.19.patch

```diff
@@ -0,0 +1,211 @@
+diff --git a/node_modules/@haqq/provider-ledger-react-native/dist/provider.js b/node_modules/@haqq/provider-ledger-react-native/dist/provider.js
```

Comment:
maybe better to fix original package?

Reply:
I need to test this branch in separate build first. After successful run, I'll move this patch into original repo.